lib/models/file-info.js

```diff
@@ -17,7 +16,7 @@ function processTemplate(content, context) {
     interpolate: /<%=([\s\S]+?)%>/g,
     escape:      /<%-([\s\S]+?)%>/g
   };
-  return template(content, options)(context);
+  return require('lodash.template')(content, options)(context);
```

Comment:
ya, i figured it didn't need to be part of the fast case, as it is only required in already slow scenarios  (blueprinting)